test/gpu/native/noGpu/basicMem.prediff

```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
 
 sed -i -e 's/0x.*/0xPREDIFFED/' $2
+sed -i -e '/<internal>/d' $2
```

Comment:
Are these indeterministic calls stuff to worry about? Do you have a guess?

Reply:
[Saw your message on slack]. It looks like "tasking layer unspecified" allocations come from qthread's internal allocations. That's as far as I am willing to investigate, and I'm fine with filtering them for the time being.